src/cftlibTest/java/uk/gov/hmcts/sptribs/cftlib/util/PlaywrightHelpers.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.sptribs.cftlib.util;
+
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.options.SelectOption;
+
+public class PlaywrightHelpers {
+    public static final Page.WaitForURLOptions LONG_WAIT = new Page.WaitForURLOptions().setTimeout(30000);
+    public static final int RETRIES = 5;
+
+    private PlaywrightHelpers() {
+    }
+
+    public static SelectOption select(String label) {
```

Comment:
I don't think this method is currently used - will it will be used with future tests?

Reply:
yes it is for future tests